vault/resource_database_secret_backend_connection.go

```diff
@@ -752,6 +754,20 @@ func connectionStringResource(config *connectionStringConfig) *schema.Resource {
 		}
 	}
 
+	if config.isCloud {
+		res.Schema["auth_type"] = &schema.Schema{
```

Comment:
Small nit, but should we maybe use constants for these fields? We can add them to the [internal/consts.go](https://github.com/hashicorp/terraform-provider-vault/blob/main/internal/consts/consts.go) file where we keep most of them?

Reply:
I think we could - it didn't match with the rest of the file so i didn't commit.